host/crypto/bcrypt/sha.c

```diff
@@ -81,3 +119,64 @@ oe_result_t oe_sha256_final(oe_sha256_context_t* context, OE_SHA256* sha256)
 done:
     return result;
 }
+
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+oe_result_t oe_sha256_save(
+    const oe_sha256_context_t* context,
+    uint32_t* internal_hash,
+    uint32_t* num_hashed)
```

Comment:
```suggestion
    uint32_t* hashed_count)
```
Avoid abbreviations like 'num'